src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBStreamTestRunner.java

```diff
@@ -250,18 +254,30 @@ void await(int timeoutSeconds) {
 
         @Override
         public int compare(MTRecord r1, MTRecord r2) {
-            if (!r1.getContext().equals(r2.getContext())) return 1;
-            if (!r1.getTableName().equals(r2.getTableName())) return 1;
-            if (!r1.getEventName().equals(r2.getEventName())) return 1;
-            if (!r1.getDynamodb().getKeys().equals(r2.getDynamodb().getKeys())) return 1;
+            if (!r1.getContext().equals(r2.getContext())) {
```

Comment:
not to be dealt with in this PR, but doesn't this violate the compare contract (unless 0 is somehow always returned)? (specifically, the requirement that sgn(compare(x, y)) = -sgn(compare(y, x)) for all valid inputs x,y (from https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html#compare-T-T-))

Reply:
I wasn't aware of that requirement.  I'm making the "it's only a test excuse" again.